{
  assertTrue(throwable instanceof TimeoutException);
  finishTest();
}
